services/notification/notifiers/mixins/message/__init__.py

```diff
@@ -275,6 +275,14 @@ def get_upper_section_names(self, settings):
         if all(x not in sections for x in headers):
             sections.insert(0, "condensed_header")
 
+        print(sections)
+        if (
+            "files" in sections
+            and "newfiles" not in sections
+            and "condensed_files" not in sections
+        ):
+            sections.insert(0, "newfiles")
```

Comment:
so we're gonna show "files" AND "newfiles" ?...

The question is: do we pop the "files" entry anywhere?
Like instead of this `if` can we do `sections.replace("files", "condensed_files")` ?

Also let's prefer `condensed_files` instead of `new_files` cause that's not new anymore

Reply:
We indeed want to show both if `files` has been selected.